src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/global_navitagtion/NavigationBar.java

```diff
@@ -18,77 +17,119 @@
 
 public class NavigationBar extends WikiBasePageObject {
 
-    @FindBy(css = "#searchInput")
-    private WebElement searchInput;
-    @FindBy(css = "#searchSubmit")
-    private WebElement searchSubmit;
-    @FindBy(css = ".autocomplete div")
-    private WebElement suggestion;
-    @FindBys(@FindBy(css = ".autocomplete div"))
-    private List<WebElement> suggestionsList;
-
-    private By jqueryAutocompleteBy = By.cssSelector("[src*='jquery.autocomplete']");
-
-    public NavigationBar(WebDriver driver) {
+	public NavigationBar(WebDriver driver) {
         super(driver);
     }
 
-    public void triggerSuggestions(String query) {
-        searchInput.click();
-        waitForElementByBy(jqueryAutocompleteBy);
-        sendKeysHumanSpeed(searchInput, query);
-        waitForElementByElement(suggestion);		
-        waitForElementByElement(suggestionsList.get(0));		
-    }
-
-    public void verifySuggestions(String suggestion) {
-        waitForElementByElement(suggestionsList.get(0));
-        String allSuggestions = "";
-        for (int i = 0; i < suggestionsList.size(); i++) {
-            allSuggestions += suggestionsList.get(i).getAttribute("title");
-        }
-        Assertion.assertStringContains(suggestion, allSuggestions);
-    }
+    @FindBy(css = "#searchInput")
+	private WebElement searchInput;
+	@FindBy(css = "#searchSubmit")
+	private WebElement searchSubmit;
+	@FindBy(css = ".autocomplete div")
```

Comment:
you have the same css in three variables. Create one final variable and use it in those places